rqt_gui_py/src/rqt_gui_py/ros_py_plugin_provider.py

```diff
@@ -72,8 +72,9 @@ def _check_for_master(self):
         self._wait_for_master_thread.start()
         self._wait_for_master_dialog = QMessageBox(QMessageBox.Question,
                                                    self.tr('Waiting for ROS master'),
-                                                   self.tr("Could not find ROS master. Either start a 'roscore' or "
-                                                           "abort loading the plugin."),
+                                                   self.tr("Could not find ROS master. Either "
+                                                           "start a 'roscore' or abort loading the"
+                                                           "plugin."),
```

Comment:
This is a good example of why aligning the content with the open parenthesis is a "bad" idea. Due to the massive indentation the usable line length is tiny which results in several wraps and hard to read content. Please wrap after `QMessageBox(` and indent the block with just 4 spaces. Also the reformatting has removed the space between the last two words "the" and "plugin".